views/AddInventory.html

```diff
@@ -30,8 +35,11 @@
 
     <div class="row form-group">
         <div class="col-sm-4">
-            <label for="location_input">Location</label>
-            <input id="location_input" class="form-control" title="The location this inventory will be stored at."></input>
+            <form>
+                        Location
+                        <input type="text" id="location_input" name="location_input" value="A-1"
```

Comment:
Weird. It doesn't do that to me in firefox or chrome. What browser/version are you using?